denizen_scripts/survival/repo_link/global_systems/chunk_claim_protection.dsc

```diff
@@ -1203,6 +1210,9 @@ claiming_protection_events:
       - stop
     - if <[group2]> != null:
       - determine cancelled
+    on entity changes block:
+      - if <yaml[claims].read[<context.location.world>.<context.location.chunk.x>.<context.location.chunk.z>]||null> != null && <context.entity.entity_type> == SILVERFISH:
```

Comment:
it's better to check if a yaml contains a path rather than checking if an entry in a yaml is invalid, or not
```suggestion
      - if <yaml[claims].contains[<context.location.world>.<context.location.chunk.x>.<context.location.chunk.z>]> && <context.entity.entity_type> == SILVERFISH:
```